backend/config.py

```
@@ -0,0 +1,21 @@
+"""
+Configuration and secrets for DIY Visual Finder
+"""
+
+# Access keys for the application
+MISTRAL_API_KEY = "sample_key"
+VOYAGE_API_KEY = "sample_key"
+JWT_SECRET = "sample_key"
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "JWT_SECRET"

(S105)